<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_passowrds</class>
 <widget class="QWidget" name="show_passowrds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>show passwords</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>271</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>271</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>271</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed Black</family>
     <pointsize>12</pointsize>
     <weight>11</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 87 12pt &quot;Inconsolata Condensed Black&quot;;
selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(85, 85, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <column>
    <property name="text">
     <string>name</string>
    </property>
    <property name="font">
     <font>
      <family>Inconsolata Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="background">
     <color alpha="0">
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="font">
     <font>
      <family>Inconsolata Condensed</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="background">
     <color alpha="0">
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="font">
     <font>
      <family>Inconsolata Condensed</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="background">
     <color alpha="0">
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="search_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed Black</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SEARCH:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="s_by_login_checkbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>search by login</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="s_by_password_checkbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>search by password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inconsolata Condensed Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Add..</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
